clean.cmd

```diff
@@ -61,9 +61,9 @@ shift /1
 goto Loop
 
 :Begin
-
-echo Running init-tools.cmd
-call %~dp0init-tools.cmd
+REM If VBCSCompiler.exe is running we need to kill it
+tasklist /fi "imagename eq VBCSCompiler.exe" |find ":" > nul
```

Comment:
Are we sure that a colon will only occur in the output if the task isn't found?  Isn't it possible that there would be an "INFO:" or "ERROR:" message even when the task is running, for example?  Perhaps a preferable implementation would be to use `find /i "VBCSCompiler.exe" > NUL` and then check for the ERRORLEVEL equaling 0, as suggested in http://stackoverflow.com/questions/162291/how-to-check-if-a-process-is-running-via-a-batch-script